README.md

```diff
@@ -85,21 +85,19 @@ let res = dynamo_client
 aws_span.end(&res);
 ```
 
-Defining a custom aws target:
+Only the following AWS targets are fully supported at the moment:
 
-```rust
-struct S3Target {}
-impl IntoAttributes for S3Target {
-    fn service(&self) -> &'static str {
-        "s3"
-    }
-
-    fn into_attributes(self, _method: &'static str) -> Vec<KeyValue> {
-        vec![semcov::AWS_S3_BUCKET.string("my_bucket")]
-    }
-}
-```
+ * DynamoDB
+ * SNS
+ * Firehose
```

Comment:
I don't want to spend too much time writing instrumentation code for all SDKs.

PRs are welcome. 😄

Reply:
This is great work!!